src/Filter/ModelFilter.php

```diff
@@ -147,6 +147,10 @@ private function buildInExpression(ProxyQueryInterface $query, string $alias, Fi
         $orX = $queryBuilder->expr()->orX();
 
         foreach ($data->getValue() as $value) {
+            if (null === $value) {
```

Comment:
Wouldn't even safer to user `if (!\is_object($value)) {` ?
<img width="387" alt="image" src="https://github.com/sonata-project/SonataDoctrineORMAdminBundle/assets/9052536/70bc227f-738e-4aaf-ab35-065f56cafd80">